app/src/main/java/org/fossasia/susi/ai/skills/skilldetails/SkillDetailsFragment.kt

```diff
@@ -188,11 +188,14 @@ class SkillDetailsFragment : Fragment(), ISkillDetailsView {
     }
 
     private fun setExamples() {
-        if (skillData.examples != null) {
+        if (skillData.examples != null && skillData.examples.isNotEmpty()) {
```

Comment:
```if (!skillData.examples.isNullOrEmpty())```

Reply:
@arundhati24 is this function available for a list type? it's for a text type IMO.